apps/dashboard/app/helpers/application_helper.rb

```diff
@@ -56,6 +56,14 @@ def fa_icon(icon, fa_style: 'fas', id: '', classes: 'app-icon', title: "FontAwes
                 title: title, "aria-hidden": true)
   end
 
+  def favicon
+    if Rails.env.production?
+      favicon_link_tag('favicon.ico', href: @user_configuration.public_url.join('favicon.ico'), skip_pipeline: true)
+    else
+      favicon_link_tag('favicon.ico')
```

Comment:
As the CI indicates, newer versions of the webdriver fail when they can't find the `favicon`.

So this adds a `favicon` for non-production environments. (It's the same favicon we use on our documentation).